documentation/book/con-kafka-connect-authentication.adoc

```diff
@@ -36,4 +40,30 @@ spec:
       certificate: public.crt
       key: private.key
   # ...
+----
+
+== SCRAM-SHA-512 authentication
+
+To use the authentication using the SCRAM-SHA-512 SASL mechanism, set the `type` property to the value `scram-sha-512`.
+SCRAM-SHA-512 authentication is using username and password to authenticate.
```

Comment:
SCRAM-SHA-512 authentication is using username and password to authenticate.
The username has to be specified in the `username` property.
The password has to be specified as a link to a `Secret` containing the password in the `passwordSecret` property. >
SCRAM-SHA-512 uses username and password to authenticate. Specify the username in the `username` property. Specify the password as a link to a `Secret` containing the password in the `passwordSecret` property.